app/controllers/api/storage_services_controller.rb

```diff
@@ -21,5 +21,12 @@ def edit_resource(type, id, data = {})
         {:task_id => storage_service.update_storage_service_queue(User.current_userid, data)}
       end
     end
+
+    def check_compliant_resources_resource(type, _id = nil, data = {})
+      data["_id"] = _id
```

Comment:
definitely want to use `id` and not `_id`

Also, do we want `data["id"]` or is that underscore correct?

Reply:
no problem changing the method param name to `id`, but changing the `data` hash key to `id` raises the following error in `api_ems_resource`: `Resource id or href should not be specified for creating a new storage_services`.

perhaps I shouldn't have used `api_ems_resource` if it's intended for creating new records only?
I used it because it appeared in the existing `validate` action that I based upon:
https://github.com/ManageIQ/manageiq-api/blob/e1980865bb2895ba3e72a9567ea2b3699583b9ba/app/controllers/api/physical_storages_controller.rb#L26
 
is there an alternative?

if `data["_id"]` can stay that way, I prefer using `api_ems_resource` since things are working right now..

thanks